Update 'Configure firmware' step (#479)
NordicSemiconductor/asset-tracker-cloud-docs: docs/firmware/aws/BuildingUsingDocker.rst

    git clone --branch |version| --single-branch \\
      https://github.com/NordicSemiconductor/asset-tracker-cloud-firmware-aws firmware

> If you haven't configured the firmware yet, follow the :ref:`configure the firmware instructions <aws-firmware-configuration>`.

Reviewer: ```suggestion
If you have not configured the firmware yet, follow the :ref:` instructions for configuring the firmware <aws-firmware-configuration>`.
```

> If you have not configured the firmware yet, follow the :ref:` instructions for configuring the firmware <aws-firmware-configuration>`.

Reviewer: An extra space
```suggestion
If you have not configured the firmware yet, follow the :ref:`instructions for configuring the firmware <aws-firmware-configuration>`.
```

> Then, move the :file:`~/nrf-asset-tracker/firmware.conf` file into ``~/nrf-asset-tracker/firmware``.

Reviewer 2: ```suggestion
Then, move the :file:`~/nrf-asset-tracker/firmware.conf` file into the :file:`~/nrf-asset-tracker/firmware` folder.
```

    git clone --branch |version| --single-branch \\
      https://github.com/NordicSemiconductor/asset-tracker-cloud-firmware-aws firmware

> Then, move the :file:`firmware.conf` file created on the :ref:`configuration instructions for the firmware <aws-firmware-configuration>` step into ``~/nrf-asset-tracker/firmware``.

Reviewer 2: ```suggestion
Then, move the :file:`firmware.conf` file created in the :ref:`firmware configuration phase <aws-firmware-configuration>` into `:file:`~/nrf-asset-tracker/firmware`.
```
Do the same changes to the other docs as well.

Reviewer 2: Right, Uma's proposals were way better than mine.

************************

The built HEX file will be located in :file:`build/zephyr/merged.hex`.

Device credentials
=====================

> Click :ref:`Create device credentials <aws-device-credentials>` to continue with the configuration or click ``Next`` to see other ``firmware`` build options. 

Reviewer 2: Hold on, where are we now? What are we clicking and where? If we refer to a doc page, we don't say click. Not that sure we need this section here or any of the docs in this PR.

Reviewer 3: I reviewed the changes, and the point is to bring users back to the Getting Started section, because the navigate outside of it for building the firmware.

************************

The built HEX file will be located in :file:`build/zephyr/merged.hex`.

Device credentials
******************

> For the device to be able to connect to the nRF Asset Tracker for AWS, now :ref:`create device credentials <aws-device-credentials>`.

Reviewer 4: ```suggestion
For the device to be able to connect to the nRF Asset Tracker for AWS, you must :ref:`create device credentials <aws-device-credentials>`.
```
